pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -214,34 +214,36 @@ func evictPods(inputPods []*v1.Pod,
 	totalCpu *float64,
 	totalMem *float64,
 	podsEvicted *int,
-	dryRun bool) {
+	dryRun bool, maxPodsToEvict int) {
 	if IsNodeAboveTargetUtilization(nodeUsage, targetThresholds) && (*totalPods > 0 || *totalCpu > 0 || *totalMem > 0) {
 		onePodPercentage := api.Percentage((float64(1) * 100) / float64(nodeCapacity.Pods().Value()))
 		for _, pod := range inputPods {
-			cUsage := helper.GetResourceRequest(pod, v1.ResourceCPU)
-			mUsage := helper.GetResourceRequest(pod, v1.ResourceMemory)
-			success, err := evictions.EvictPod(client, pod, evictionPolicyGroupVersion, dryRun)
-			if !success {
-				glog.Warningf("Error when evicting pod: %#v (%#v)", pod.Name, err)
-			} else {
-				glog.V(3).Infof("Evicted pod: %#v (%#v)", pod.Name, err)
-				// update remaining pods
-				*podsEvicted++
-				nodeUsage[v1.ResourcePods] -= onePodPercentage
-				*totalPods--
-
-				// update remaining cpu
-				*totalCpu -= float64(cUsage)
-				nodeUsage[v1.ResourceCPU] -= api.Percentage((float64(cUsage) * 100) / float64(nodeCapacity.Cpu().MilliValue()))
-
-				// update remaining memory
-				*totalMem -= float64(mUsage)
-				nodeUsage[v1.ResourceMemory] -= api.Percentage(float64(mUsage) / float64(nodeCapacity.Memory().Value()) * 100)
-
-				glog.V(3).Infof("updated node usage: %#v", nodeUsage)
-				// check if node utilization drops below target threshold or required capacity (cpu, memory, pods) is moved
-				if !IsNodeAboveTargetUtilization(nodeUsage, targetThresholds) || (*totalPods <= 0 && *totalCpu <= 0 && *totalMem <= 0) {
-					break
+			if *podsEvicted+1 <= maxPodsToEvict {
```

Comment:
Nit: I prefer more:

```go
if *podsEvicted+1 > maxPodsToEvict {
     break
}
```

rather than to move entire block. It looks better in the diff and gives assumptions valid till the end of the for block.

Reply:
Sure. I can do that.